docs/docs/references/cli/exec/index.md

```diff
@@ -0,0 +1,166 @@
+---
+sidebar_label: exec
+---
+# Command: `exec`
+
+## Description
+
+The `bacalhau exec` command allows for the specification of jobs to be executed from the command line,
+without the need for a job specification file (see [job run](../job/run/)). 
+
+## Usage
+
+```shell
+bacalhau exec [flags] [job-type] arguments
+```
+
+## Flags
+
+- `-h`, `--help`:
+    - Description: Displays help information for the `exec` sub-command.
+
+- `--code`: 
+    - Includes the specified code in the job. This can be a single file, or a directory containing many files.  There is a limit of 10Mb on the size of the uploaded code.
+
+- `-f`, `--follow`:
+    - Description: If provided, the command will continuously display the output from the job as it runs.
+
+- `--wait`
+	- Description: Wait for the job to finish. Use --wait=false to return as soon as the job is submitted.
+
```

Comment:
While my opinion deviates from our usual practice - I'll share it anyways: I have a predisposition against boolean flags that are enabled by default because using them often requires explicitly disabling them. 

Consider the difference between using a flag like `exec --wait=false <commands>` vs. `exec --no-wait <commands>`. Preference for the latter since the presence of the `--no-wait` flag implies an active choice to skip waiting, with its default state being inactive (false). This means if the flag is used, it's assumed to be true. However, I agree that using negative terms in flags can sometimes lead to confusion due to the inherent double negatives.

Reply:
Totally agree, and we should remove the --wait flags from v2 of the CLI